website/.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Production
 /build
+/out
```

Comment:
Used by TypeScript's project mode. `noEmit` will still generate metadata to cache types.